storage_v2/factoryadapter/factory.go

```diff
@@ -35,8 +35,12 @@ func (f *Factory) Close(_ context.Context) error {
 }
 
 // CreateTraceReader implements spanstore.Factory.
-func (*Factory) CreateTraceReader() (spanstore.Reader, error) {
```

Comment:
can we move this change to another PR and merge it first?

Reply:
btw this would be a good use case for samling

Reply:
@yurishkuro https://github.com/jaegertracing/jaeger/pull/6221